releases.yml

```diff
@@ -275,10 +284,15 @@ patches:
 
   # CVSS Score >= 6.0
 
-  # CVSS Score 6.0, < k8s1.16, https://www.cvedetails.com/cve/CVE-2020-8559/
+  # CVSS Score 6.0, < k8s1.21, https://www.cvedetails.com/cve/CVE-2020-8554/
+  - name: CVE-2020-8554
```

Comment:
Why should we add `CVE-2020-8554` here? We have `CVE-2020-8554.1.19` add below? I can't quite understand the reason.

Reply:
Add the original link before patch modification to facilitate searching later